assets/css/base.css

```diff
@@ -0,0 +1,221 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+
+@font-face {
+  font-family: 'Cocogoose';
+  src:
+    url('cocogoose_pro-trial-webfont.woff2') format('woff2'),
+    url('cocogoose_pro-trial-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+ul li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
+  width: 100%;
+  height: 85px;
+  background: #fff 0% 0% no-repeat padding-box;
+  box-shadow: 0 3px 15px #00000029;
+  opacity: 1;
+  padding: 10px 140px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-logo {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  text-align: center;
+  background: transparent url('') 0% 0% no-repeat padding-box;
+  opacity: 1;
+}
+
+.nav-logo a {
+  font-family: 'Cocogoose', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 35px;
+  color: #272a31;
+}
+
+nav .nav-links-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav .nav-links-list li a {
+  text-align: center;
+  font: normal normal 500 18px/20px 'Lato', sans-serif;
+  letter-spacing: 1px;
+  color: #272a31;
+  opacity: 1;
+}
+
+.nav-links-list li {
+  justify-content: center;
+  align-items: center;
+  margin-right: 40px;
+}
+
+#top-bar {
+  background: #272a31;
+  width: 100%;
+  padding: 15px 140px;
+}
+
+#top-bar ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+#top-bar ul li {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 15px;
+}
+
+#top-bar ul li a {
+  text-align: center;
+  font: normal normal 400 12px/15px 'Lato', sans-serif;
+  letter-spacing: 0;
+  color: #fff;
+  opacity: 1;
+}
+
+#campain {
+  margin-left: 20px;
+  margin-right: 0;
+  border: 3px solid #ec5242;
+  padding: 12px;
+}
+
+#campain-link {
+  color: #ec5242;
+}
+
+.burger {
+  display: none;
+  cursor: pointer;
+}
+
+.burger div {
+  width: 32px;
+  height: 6px;
+  background-color: #272a31;
+  margin: 5px;
+  transition: all 0.3s ease;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly set up your app to be designed using the mobile-first approach like what the first reviewer said. Mobile-first is one of the main principles in web development and it's really important. Please keep in mind that this is one of the requirements in one of the milestones of the project. 
![image](https://user-images.githubusercontent.com/62290545/169344334-db3d9ea2-15ab-4e0e-973e-bc090957c788.png)

Reply:
@ShadyShawkat I think, if the project is working as expected, this should be an optional requirement, since its only a recommendation

Reply:
Hey @NduatiKagiri, The previous review marked this as a required change, and since this is in the project's requirements, I can't disregard it.